pipeline/conftest.py

```diff
@@ -20,7 +20,7 @@ def example_detailed_event():
 def expected_columns():
     """List of columns expected from sql query to database."""
     return [
-        "earthquake_id", "magnitude", "time", "updated", "longitude", "latitude",
+        "earthquake_id`", "magnitude", "time", "updated", "longitude", "latitude",
```

Comment:
Why is there a "`" here?
```suggestion
        "earthquake_id", "magnitude", "time", "updated", "longitude", "latitude",
```